src/main/java/mate/academy/internetshop/lib/Injector.java

```diff
@@ -1,35 +1,99 @@
 package mate.academy.internetshop.lib;
 
-import core.basesyntax.controller.ConsoleHandler;
-import core.basesyntax.dao.BetDao;
-import core.basesyntax.dao.BetDaoImpl;
-import core.basesyntax.dao.UserDao;
-import core.basesyntax.dao.UserDaoImpl;
-import core.basesyntax.factory.BetDaoFactory;
-import core.basesyntax.factory.UserDaoFactory;
-
 import java.lang.reflect.Field;
+import mate.academy.internetshop.Factory;
+import mate.academy.internetshop.Main;
+import mate.academy.internetshop.dao.BucketDao;
+import mate.academy.internetshop.dao.ItemDao;
+import mate.academy.internetshop.dao.OrderDao;
+import mate.academy.internetshop.dao.UserDao;
+import mate.academy.internetshop.dao.impl.BucketDaoImpl;
+import mate.academy.internetshop.dao.impl.ItemDaoImpl;
+import mate.academy.internetshop.dao.impl.OrderDaoImpl;
+import mate.academy.internetshop.dao.impl.UserDaoImpl;
+import mate.academy.internetshop.service.BucketService;
+import mate.academy.internetshop.service.ItemService;
+import mate.academy.internetshop.service.OrderService;
+import mate.academy.internetshop.service.UserService;
+import mate.academy.internetshop.service.impl.BucketServiceImpl;
+import mate.academy.internetshop.service.impl.ItemServiceImpl;
+import mate.academy.internetshop.service.impl.OrderServiceImpl;
+import mate.academy.internetshop.service.impl.UserServiceImpl;
 
 public class Injector {
 
-    public static void injectDependency() throws IllegalAccessException {
-        Class<ConsoleHandler> consoleHandlerClass = ConsoleHandler.class;
-        Class<BetDaoImpl> betDaoImplClass = BetDaoImpl.class;
+    public static void injectDependencies() throws IllegalAccessException {
+        inject(UserServiceImpl.class.getDeclaredFields());
+        inject(OrderServiceImpl.class.getDeclaredFields());
+        inject(ItemServiceImpl.class.getDeclaredFields());
+        inject(BucketServiceImpl.class.getDeclaredFields());
+        inject(Main.class.getDeclaredFields());
+    }
+
+    private static void inject(Field[] fields) throws IllegalAccessException {
         Class<UserDaoImpl> userDaoImplClass = UserDaoImpl.class;
-        Field[] consoleHandlerFields = consoleHandlerClass.getDeclaredFields();
-        for (Field field : consoleHandlerFields) {
+        Class<OrderDaoImpl> orderDaoImplClass = OrderDaoImpl.class;
+        Class<ItemDaoImpl> itemDaoImplClass = ItemDaoImpl.class;
+        Class<BucketDaoImpl> bucketDaoImplClass = BucketDaoImpl.class;
+        Class<UserServiceImpl> userServiceImplClass = UserServiceImpl.class;
+        Class<OrderServiceImpl> orderServiceImplClass = OrderServiceImpl.class;
+        Class<ItemServiceImpl> itemServiceImplClass = ItemServiceImpl.class;
+        Class<BucketServiceImpl> bucketServiceImplClass = BucketServiceImpl.class;
+        for (Field field : fields) {
             if (field.getDeclaredAnnotation(Inject.class) != null
```

Comment:
we can make one if statement general:
if(field.getDeclaredAnnotation(Inject.class) != null){
if(...){...}
...
}